js/ui/marker.js

```diff
@@ -44,6 +44,7 @@ Marker.prototype = {
         this._map = map;
         map.getCanvasContainer().appendChild(this._element);
         map.on('move', this._update);
+        map.on('moveend', this._update);
         this._update();
```

Comment:
As the PR stands, it does round coords on initialization (only keeps un-rounded coords on `move`). We can add this option in the interest of more explicit/understandable code 👍